datafusion/optimizer/src/single_distinct_to_groupby.rs

```diff
@@ -98,6 +98,7 @@ impl OptimizerRule for SingleDistinctToGroupBy {
                 aggr_expr,
                 schema,
                 group_expr,
+                ..
```

Comment:
this is unfortunate (previously when new fields were added, the compiler would tell you all locations that might need to be changed. I realize this is a consequence of marking the structs `non_exhaustive`